jms/src/test/scala/akka/stream/alpakka/jms/scaladsl/JmsConnectorsSpec.scala

```diff
@@ -782,5 +785,92 @@ class JmsConnectorsSpec extends JmsSpec {
 
       result.futureValue should ===(input)
     }
+
+    "publish and consume strings through a queue with multiple sessions" in withServer() { ctx =>
+      //#connection-factory
```

Comment:
You need to remove the copied `//#` as that marks what Paradox cuts out for the documentation.
It would be very useful if you added a section on the producer settings in the documentation.

Reply:
I see, I wasn't sure what `//#` does. I'll add producer settings documentation as well.